src/main/java/roomescape/reservation/controller/ReservationController.java

```diff
@@ -0,0 +1,47 @@
+package roomescape.reservation.controller;
+
+import java.net.URI;
+import java.util.List;
+
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import roomescape.reservation.dto.ReservationRequestDto;
+import roomescape.reservation.dto.ReservationResponseDto;
+import roomescape.reservation.service.ReservationService;
+
+@RestController
+@RequestMapping("/reservations")
+public class ReservationController {
+    private final ReservationService reservationService;
+
+    public ReservationController(final ReservationService reservationService) {
+        this.reservationService = reservationService;
+    }
+
+    @GetMapping
+    public ResponseEntity<List<ReservationResponseDto>> findAll() {
+        List<ReservationResponseDto> responseDtoList = reservationService.findAll().stream()
```

Comment:
```suggestion
        List<ReservationResponseDto> responseDtos = reservationService.findAll().stream()
```
자료구조를 변수명에 나타내는것은 안좋은 방법일것 같아요!